<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeticketdialog</class>
 <widget class="QDialog" name="changeticketdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>changeticket</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="canclebtn">
       <property name="text">
        <string>cancle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="changebtn">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="initialEtlineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="initialBtlineEdit"/>
       </item>
       <item row="6" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="etlable">
         <property name="text">
          <string>到站时间</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>想要修改的车次</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="initialBalineEdit"/>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="idlabel">
         <property name="text">
          <string>列车号</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="initialEalineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="initialIdlineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ealabel">
         <property name="text">
          <string>终点站</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="btlabel">
         <property name="text">
          <string>出发时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="balabel">
         <property name="text">
          <string>始发站</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="finalPrlineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="balabel_2">
         <property name="text">
          <string>始发站</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="ticketlabel_2">
         <property name="text">
          <string>车票数</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="btlabel_2">
         <property name="text">
          <string>出发时间</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>票价</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="finalTicketlineEdit"/>
       </item>
       <item row="7" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="etlable_2">
         <property name="text">
          <string>到站时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="finalBalineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="finalEtlineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="finalEalineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ealabel_2">
         <property name="text">
          <string>终点站</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="finalBtlineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>修改为</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
